shell/assets/translations/en-us.yaml

```diff
@@ -4134,6 +4134,9 @@ plugins:
         image:
           label: Catalog Image Reference
           placeholder: "e.g. hub.docker.io/example-org/my-image:latest"
+        imagePullSecret:
+          label: Image Pull Secret
```

Comment:
Should it be "Image Pull Secrets"?

Reply:
ahh, thought it was multiple, but the `:multiple="true"` was clear as day. good catch!